io.openems.edge.evcs.spelsberg.smart/src/io/openems/edge/evcs/spelsberg/smart/Config.java

```diff
@@ -0,0 +1,39 @@
+package io.openems.edge.evcs.spelsberg.smart;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(//
+		name = "EVCS Spelsberg SMART", //
+		description = "Implements the Spelsberg SMART electric vehicle charging station")
+@interface Config {
+
+	@AttributeDefinition(name = "Component-ID", description = "Unique ID of this Component")
+	String id() default "evcs0";
+
+	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
+	String alias() default "";
+
+	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	boolean enabled() default true;
+
+	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus bridge.")
+	String modbus_id() default "modbus0";
+
+	@AttributeDefinition(name = "Modbus Unit-ID", description = "The Unit-ID of the Modbus device.")
+	int modbusUnitId() default 1;
+
+	@AttributeDefinition(name = "Modbus target filter", description = "This is auto-generated by 'Modbus-ID'.")
+	String Modbus_target() default "(enabled=true)";
+
+	@AttributeDefinition(name = "Debug Mode", description = "Activates the debug mode")
+	boolean debugMode() default false;
+
+	@AttributeDefinition(name = "Minimum hardware current", description = "Minimum current of the Charger in mA.", required = true)
+	int minHwCurrent() default 6000;
+
+	@AttributeDefinition(name = "Maximum hardware current", description = "Maximum current of the Charger in mA.", required = true)
+	int maxHwCurrent() default 16000;
```

Comment:
I have updated the maxHwCurrent limit to 32kW for now.
Is that enough to workaround the known issue? 
Is there a bug ticket so that i can track the progress?

Reply:
Currently there is no bug ticket, 32000 is the workaround for now.